lazy-pages/src/common.rs

```diff
@@ -125,7 +127,7 @@ impl LazyPagesContext {
 
 pub(crate) type Costs = [u64; CostNo::Amount as usize];
 pub(crate) type GlobalNames = Vec<LimitedStr<'static>>;
-pub type PageSizes = [NonZeroU32; PageSizeNo::Amount as usize];
+pub(crate) type PageSizes = [NonZeroU32; SIZES_AMOUNT];
```

Comment:
mark SIZES_AMOUNT as sensitive var since it may affect lazy pages in runtime

Reply:
SIZES_AMOUNT does not used in runtime, so it cannot affect it directly

Reply:
its used in threadlocal LAZY_PAGES_CONTEXT, so why isn't sensitive? moreover I ask just for small comment

Reply:
LAZY_PAGES_CONTEXT is inner static variable which is part of binary part only

Reply:
I cannot understand why we need then append this as sensitive and all other items ?

Reply:
Ok, comment was added